cmd/local-locker.go

```diff
@@ -145,32 +144,28 @@ func (l *localLocker) removeEntry(name, uid string, lri *[]lockRequesterInfo) bo
 }
 
 func (l *localLocker) RLock(ctx context.Context, args dsync.LockArgs) (reply bool, err error) {
-	select {
-	case <-ctx.Done():
-		return false, ctx.Err()
```

Comment:
shouldn't context cancellation be handled here?

Reply:
No actually @poornas this case will never happen it's frivolous to add this code

Reply:
There is no real wait on the context here, that code block is never executed in its lifetime. The main issue that we had to handle was stale locks - i.e owner is dead and we have no idea who owns the lock. 

Since this ownership is fixed adding context here is meaningless, its only meaning over the network.

Reply:
can we just remove ctx from args then as it is unused?

Reply:
Its an interface, needed by the network side of things @poornas - here its just a local locker